README.md

```diff
@@ -170,9 +170,10 @@ main()
 
 ```dockerfile
 FROM keybaseio/client:nightly-node
-WORKDIR /app
-COPY . /app
+WORKDIR /home/keybase
```

Comment:
```suggestion
RUN mkdir /app && chown keybase:keybase /app
WORKDIR /app
```

Reply:
Yeah, you are right with "Keybase would dump all of its cache and config files in that volume", your suggestion is better.